### PR TITLE
ng_sixlowpan: write protect pkt

### PR DESCRIPTION
**Pull request by authmillenon**
Fixes some issues with the network stack, but not all. Carry on searching.

### CODE REVIEW

File: sys/net/network_layer/ng_sixlowpan/ng_sixlowpan.c [hunk 1]
```diff
@@ -59,6 +59,21 @@ void _receive(ng_pktsnip_t *pkt)
     uint8_t *dispatch;
     ng_netreg_entry_t *entry;
 
+    /* seize payload as a temporary variable */
+    payload = ng_pktbuf_start_write(pkt);   /* need to duplicate since pkt->next
```
Comment (PeterKietzmann) [inline sys/net/network_layer/ng_sixlowpan/ng_sixlowpan.c]: So mainly you need this outside the `dispatch[0] == NG_SIXLOWPAN_UNCOMPRESSED` dispatch, right? The rest seems really redundant to me, comparing lines  79-96. I just had a quick look, please correct me if I'm wrong!

Comment (authmillenon) [inline sys/net/network_layer/ng_sixlowpan/ng_sixlowpan.c]: No `ng_pktbuf_start_write()` just duplicates this one snip

Comment (authmillenon) [inline sys/net/network_layer/ng_sixlowpan/ng_sixlowpan.c]: This is why payload needs to be duplicated too later on

**Comment by PeterKietzmann** [comment]
ACK

**Comment by PeterKietzmann** [comment]
when changes from phone call are adapted :-)

**Comment by PeterKietzmann** [comment]
ACK when travis is happy

**Pull request merged**